src/utils/whois.js

```diff
@@ -1,29 +1,36 @@
-const rdapLookup = module.exports.rdapLookup = async query => {
+const rdapLookup = async query => {
     const resp = await fetch(`https://rdap.cloud/api/v1/${query}`);
     const rawData = await resp.json().catch(() => false);
     const data = rawData?.results?.[query];
 
     // Ensure the data is there
-    if (!data || !data.success || !data.data)
-        return false;
+    if (!data || !data.success || !data.data) return false;
+
 
     // Utils to find the data
     const uniqueCommaSep = arr => [...new Set(arr)].join(', ');
 
-    const findEntities = name => data.data?.entities?.filter(entity =>
-        entity.roles.map(role => role.trim().toLowerCase()).includes(name));
+    const findEntities = name => data.data?.entities?.filter(entity => entity.roles
+        .map(role => role
+            .trim()
+            .toLowerCase())
+        .includes(name));
 
-    const findEntityName = name => uniqueCommaSep(findEntities(name).map(entity =>
-        entity?.vcardArray?.[1].find(card => card[0] === 'fn')?.[3] || entity?.handle));
+    const findEntityName = name => uniqueCommaSep(findEntities(name)
+        .map(entity => entity?.vcardArray?.[1]
+            .find(card => card[0] === 'fn')?.[3] || entity?.handle));
 
-    const findEntityEmail = name => uniqueCommaSep(findEntities(name).map(entity =>
-        entity?.vcardArray?.[1].find(card => card[0] === 'email')?.[3]));
+    const findEntityEmail = name => uniqueCommaSep(findEntities(name)
+        .map(entity => entity?.vcardArray?.[1]
+            .find(card => card[0] === 'email')?.[3]));
 
-    const findEvent = name =>
-        data.data?.events?.find(event => event.eventAction.trim().toLowerCase() === name)?.eventDate;
+    const findEvent = name => data.data?.events?.find(event => event
+        .eventAction
+        .trim()
+        .toLowerCase() === name)?.eventDate;
```

Comment:
Wrapping these seems very pointless?

Reply:
The trim and lowercase have to be wrapped otherwise it exceeds max-len